modules/mineapi/src/commands/tpss.ts

```diff
@@ -0,0 +1,31 @@
+// Import Types
+import { Command } from "@spookelton/wrapperHelpers/types";
+
+import { helpHelper } from "@spookelton/wrapperHelpers/modul";
+
+// Import helper functions for tps handling.
+import { runTpsCommand, tpsToColor, formatDimension } from "../lib/tps"
+
+
+export const tpss: Command = async (message) => {
+    // Run the tps command, and wait for a response from server console.
+	const result = await runTpsCommand();
+
+	// Construct the message containing tps info
+	return {
+        discord: {
+            embeds: [{
+                title: "Server TPS:",
+                description:
+                    result.dimensions.map(formatDimension).join("\n") +
+                    `\n\n**Overall**:\n TPS: \`${result.overall.tps}\`, Tick Time: \`${result.overall.tickTime}\` ms`,
+                color: tpsToColor(result.overall.tps),
+                timestamp: Date.now(),
+            }]
+        }
+    };
+};
```

Comment:
Just want to make sure, is it ok if I don't return anything for minecraft or console, since executing the command should return an output in console.

Reply:
Not returning anything for a specific output type will just result in no output when executing the command. So you dont have to. Afik console/ingame using ```/forge tps``` is better anyway so in this case its fine.